gradle-plugin/README.md

```diff
@@ -210,20 +210,26 @@ emerge {
   ..
 
   snapshots {
+    buildType.set("snapshots") // Build type to use for grouping builds in the Emerge dashboard
+
+    fromMainSourceSet.set(
+      true
+    ) // Generate composable previews snapshots from the main sourceSet, defaults to `false`
```

Comment:
nit: this naming makes it feel like an either/or situation (we either search from `main` or `androidTest`, not both), but from the PR description it looks like it should still be possible to define some previews inside `androidTest` since we move the generated main sourceSet files to `androidTest`. I like the `includeFromMainSourceSet` naming since it signals that we merge the two.

Reply:
Like it, updated!